numpy/polynomial/chebyshev.pyi

```diff
@@ -80,19 +80,19 @@ __all__ = [
     "chebinterpolate",
 ]
 
-_SCT = TypeVar("_SCT", bound=np.number[Any] | np.object_)
-def _cseries_to_zseries(c: npt.NDArray[_SCT]) -> _Series[_SCT]: ...
-def _zseries_to_cseries(zs: npt.NDArray[_SCT]) -> _Series[_SCT]: ...
+_ScalarT = TypeVar("_ScalarT", bound=np.number[Any] | np.object_)
```

Comment:
There's another typevar with the same `bound` in `_core/fromnumeric.pyi`. Syncing their names would make things a bit more consistent.